kotlin-runtime/ftl-plugin/build.gradle.kts

```diff
@@ -0,0 +1,34 @@
+plugins {
+  kotlin("jvm") version "1.9.0"
+  id("java-gradle-plugin")
+}
+
+repositories {
+  // Use Maven Central for resolving dependencies.
+  mavenCentral()
+}
+
+group = "xyz.block"
+version = "1.0-SNAPSHOT"
+
+gradlePlugin {
+  plugins {
+    create("ftl") {
+      id = "xyz.block.ftl"
+      displayName = "FTL"
+      implementationClass = "xyz.block.ftl.gradle.FTLPlugin"
+      description = "generate ftl stubs"
+    }
+  }
+}
+
```

Comment:
Add this so it doesn't install the gradle wrapper:

```
// Disable gradlew because we use a Hermit-provided gradle.
tasks.findByName("wrapper")?.enabled = false
```

Then delete `rm -rf gradlew* gradle` from the PR.